testrunner-toolkit/cypress/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cypress-1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "open": "npx cypress open",
+    "test.local": "npx cypress run",
+    "test.local.headfull": "npx cypress run --headed",
+    "test.docker.sauce": "npx saucectl run --test-env docker",
+    "test.sauce.ex1.eu": "DATE=\\\"$(date)\\\" npx saucectl run -c ./.sauce/config-ex1.yml --test-env sauce",
```

Comment:
You can remove all `--test-env sauce` as mentioned above

Reply:
Done